app/routes/index.js

```diff
@@ -7,10 +7,14 @@ const fxPromo = require('../templates/fxPromo');
 
 const app = choo();
 
+function showBanner(state) {
+  return !(state.promo && state.route.startsWith('/unsupported/'));
```

Comment:
The parameters look good, but the boolean logic isn't quite what we want. For example this return `true` when both are false.

The truth table we want is:

show | promo | unsupported
------|-------|-------------
true | true | false
false | true | true
false | false | true
false | false | false

Reply:
I have made the changes.